File: 2490-Circular-Sentence/without_splitting.js
// https://leetcode.com/problems/circular-sentence/submissions/1053730685/

/**
 * @param {string} sentence
 * @return {boolean}
 */
var isCircularSentence = function(sentence) {
    for(let i = 0; i < sentence.length; ++i){
        if(sentence[i] === ' ' && sentence[i - 1] !== sentence[i + 1]){
            return false
        }
    }
    return sentence[0] === sentence[sentence.length - 1]
};

File: 2490-Circular-Sentence/without_splitting.py
# https://leetcode.com/problems/circular-sentence/submissions/1053728456/

class Solution:
    def isCircularSentence(self, sentence: str) -> bool:
        for i in range(len(sentence)):
            if sentence[i] == ' ' and sentence[i-1] != sentence[i+1]:
                return False
        return sentence[0] == sentence[-1]
